12-Gestion des API.md

```diff
@@ -4,12 +4,12 @@
 
 ### Nommage
 
-les dépôts Github hébergeant les API doivent être nommés en respectant la nomenclature : <xxx>-api.
+les dépôts Github hébergeant les API doivent être nommés en respectant la nomenclature : ```<xxx>-api```.
```

Comment:
pour info pour citer du code inclus dans une phrase, c'est plutôt des simple quote inversées qu'il faut utiliser et pas des triples quotes inversées

cf https://docs.github.com/fr/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code

Reply:
Je viens d'apporter la correction sur la triple cote. J'en ai profité pour modifier deux autres endroits problématiques.
Merci !